### ISSUE TITLE
Generate Formal Employment Contracts for Nelson & Inês

### ISSUE THREAD

**Issue by ghost** [dwyl/hq#241]
Generate Formal Employment Contracts for Nelson & Inês

**Comment by markwilliamfirth** [comment]
@iteles [link](https://docs.google.com/document/d/1Zzdn2pA29iLyduuogudTlss-loDPiCtfl_5iWhoKfpA/edit)

@nelsonic [link](https://docs.google.com/document/d/1FEYAw36LRc7MURdvYc0UQK3HwBSyglbysLTtY6AUI-k/edit)

Please can you write your `Normal duties` into your contract?

**Comment by nelsonic** [comment]
Hi @markwilliamfirth, I _agree_ that is _high_ priority, but don't feel like it has to be done today.
Can you give us a specific timeline for when you would _like_ to get this done? _thanks_!

**Comment by markwilliamfirth** [comment]
@nelsonic this week?

**Comment by markwilliamfirth** [comment]
@nelsonic @iteles this wasn't done last week

**Comment by iteles** [comment]
Nope. No it wasn't. Let's talk about it in our Monday prioritisation meeting?

**Comment by markwilliamfirth** [comment]
@iteles could you and nelson write your draftsin preparation and we can review them in the meeting?

**Comment by iteles** [comment]
Sorry, I have three project backlogs to review in their entirety before Monday morning, I'm going to find this difficult 😭

**Comment by markwilliamfirth** [comment]
ok I don't think we should write them in the meeting though - not a good use of collective time - can we have them by this friday @nelsonic @iteles ?

**Comment by iteles** [comment]
@markwilliamfirth Completely agree, I just meant discuss

**Comment by nelsonic** [comment]
I _agree_ that this needs to be done. Contracts are _important_ for accountability.

So far my "role" in the company has ben "_cash cow_", and I think I've achieved it _rather_ well. ✅ 
But for "_official_" (_i.e. contract_) reasons it does "make sense" to have a more descriptive role.

Please just write down the role/responsibilities of a Lead Developer, Product Manager, "CTO" or what ever title I need to have for "official reasons" and _send_ me a contract to sign. 

I don't think I need to be _assigned_ to the task. so I'm ***unassigning*** myself.

My _exclusive_ focus for the foreseeable future is writing/reviewing the code for the client projects and _eventually_ for the product(s).

Thanks! ❤️  ✅  🚀

**Comment by nelsonic** [comment]
@markwilliamfirth / @iteles can you please review the responsibilities I've added to: https://docs.google.com/document/d/1FEYAw36LRc7MURdvYc0UQK3HwBSyglbysLTtY6AUI-k
I'm happy to print this off and sign it today so we can get it scanned.

**Comment by markwilliamfirth** [comment]
Mark to review

**Comment by iteles** [comment]
@markwilliamfirth Added my responsibilities for review over the weekend and discussed.

Also added comments to the document around:
+ 'normal hours of work' and the need for flexible working hours
+ A limit (or lack thereof) on number of days holiday in a year
+ Intellectual property portion of contract requires rewording (this might require some thought!)

**Comment by markwilliamfirth** [comment]
@iteles I've responded to comments on the documents:

[employment template](https://docs.google.com/document/d/1MT42M250Sdi1rbyUiRDh4BMkFn12ZLj_L1Cl8K9ASxM/edit)

[ines](https://docs.google.com/document/d/1Zzdn2pA29iLyduuogudTlss-loDPiCtfl_5iWhoKfpA/edit)

**Comment by markwilliamfirth** [comment]
@nelsonic I have commented [here](https://docs.google.com/document/d/1FEYAw36LRc7MURdvYc0UQK3HwBSyglbysLTtY6AUI-k/edit)

**Comment by nelsonic** [comment]
@markwilliamfirth thanks. I think we're still going to make a final update on "_vesting_" today, but once that's done we should print & sign. 👍

**Comment by markwilliamfirth** [comment]
@nelsonic @iteles yep - I think they're good to go now

**Comment by markwilliamfirth** [comment]
@nelsonic and @iteles to print and sign 2 copies and then either countersign each other's or hand to myself to countersign.

Needs to be wet signature - we can't e-sign as you have power of attorney as a Director as well as other reasons.

>If, however, there’s a requirement for the employment contract to be executed as a deed don’t use an electronic signature at all. That’s because the employee’s signature must be witnessed by a person who’s in the same physical location at the time the employee signs the employment contract. Thankfully, this rule only applies in a limited number of situations, e.g. where the employment contract includes a power of attorney, so it won’t apply to the vast majority of employment contracts.

**Comment by iteles** [comment]
Thanks for looking this up 👍 We'll print them and we can ignore the e-signed ones then.

**Comment by nelsonic** [comment]
@markwilliamfirth any luck scanning in the _signed_ paper copies of the contracts? 🤔

**Comment by markwilliamfirth** [comment]
These have been scanned and saved into drive. These will need to be revisited due to the decision to start official employment on April 6 and after #324 has been resolved

**Comment by iteles** [comment]
Nothing was done about these, should be a quick change on the employment date. This _should_ have been minuted in Board meeting minutes but I can't remember reading this there off the top of my head so will need to check.

Role descriptions are fluid in a start-up and contract role descriptions should be a starting point as per #324.

**Comment by iteles** [comment]
No longer relevant.
